packages/node/src/sdk.ts

```diff
@@ -152,3 +144,34 @@ export async function close(timeout?: number): Promise<boolean> {
   }
   return Promise.reject(false);
 }
+
+/**
+ * A function that returns a Sentry release string dynamically from env variables
+ */
+export function getSentryRelease(): string {
+  // Fetches the variable that sentry-webpack-plugin injects
+  const global = getGlobalObject<Window>();
+  let globalSentryRelease = undefined;
+  if (global.SENTRY_RELEASE && global.SENTRY_RELEASE.id) {
+    globalSentryRelease = global.SENTRY_RELEASE.id;
+  }
+
+  const sentryRelease = JSON.stringify(
+    // Always read first as Sentry takes this as precedence
+    process.env.SENTRY_RELEASE ||
```

Comment:
If `process.env.SENTRY_RELEASE` is always first, then there's no need to check all the globals and even fetch it on line 153.

We can do a short-circuit at the very beginning of the function:

```js
if (process.env.SENTRY_RELEASE) {
  return process.env.SENTRY_RELEASE;
}

// ... the rest of checks
```